### PR TITLE
WizardController: return results in jump and replace

### PR DESCRIPTION
**Pull request by d-loose**

### CODE REVIEW

**Review by jpnurmi — DISMISSED** [review]
Thanks, LGTM :+1:

File: lib/src/controller.dart [hunk 1]
```diff
@@ -110,7 +110,7 @@ class WizardController extends ChangeNotifier {
 
   /// Requests the wizard to replace the current page with the next one.
   /// Optionally, `arguments` can be passed to the next page.
-  void replace({Object? arguments}) async {
```
Comment (jpnurmi) [inline lib/src/controller.dart:113]: wait a sec, did we mix up something with the result and argument types? the type of arguments (input) passed to the next route, and the result (output) returned from there shouldn't be the same type.

Comment (d-loose) [reply lib/src/controller.dart:113]: Should be sorted out now :)

**Review by jpnurmi — APPROVED** [review]
LGTM :+1:

**Pull request merged**